src/components/button-icon/types.ts

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from "react";
+
+export enum BUTTON_ICON_KIND {
```

Comment:
There is no toggle button on the template, but i think it is a case as well. We need to add toggle icon as well.

Reply:
I mean, this is because toggle kind was added later and icon button was not updated probably

Reply:
I don't have toggle case now so i'll skip it now